src/bin/rewrite_app_repos.sh

```diff
@@ -97,13 +98,29 @@ for plugin_branch_env_var in "${plugin_branch_env_vars[@]}"; do
       continue
     fi
 
-    # We don't need to build the assets for tags anymore, as we made those work with github releases.
-    # Branches still need to be built at this point for now.
-    # We know if it's a branch when the prefix was present, so $branch should differ from $plugin_version only in that case.
+    # We don't need to build the assets for tags and test instances, both are zip files
+    # Test instances use "7" as a version number
+    if [ "${plugin_version}" == "7" ]; then
```

Comment:
Maybe we should break up the file while making these changes? It's already doing quite a few different things.

Reply:
Though we don't have to do that if it would take too much time while we're waiting for a fix for the plugin. So just a suggestion.

Reply:
> Maybe we should break up the file while making these changes? It's already doing quite a few different things.

It's a big script indeed, but we refactor this when we start cleaning up all of these scripts.